easybuild/framework/easyblock.py

```diff
@@ -315,7 +317,10 @@ def fetch_patches(self, list_of_patches, extension=False, checksums=None):
                         copy_file = True
                     suff = patch_entry[1]
                 else:
-                    self.log.error("Wrong patch specification '%s', only int and string are supported as second element!" % patch_entry)
+                    self.log.error(
+                        "Wrong patch specification '%s', only int and string are supported as second element!",
```

Comment:
please collapse with line above, cfr. line 307?

if you need to, shorten the message (e.g. using "int/string", "2nd")